Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/dropdown/Dropdown.tsx

```diff
@@ -1,60 +1,105 @@
-import { MenuItem, Select } from '@mui/material';
+import { useEffect, useState } from 'react';
+import { Autocomplete, Popper, TextField } from '@mui/material';
+import { IDropdownItem } from 'src/common/types';
 
 import useTheme from '../../../hooks/use-theme';
 
+// eslint-disable-next-line css-modules/no-unused-class
```

Comment:
Because of the way the styles are applied, they are not applied with the convention of `styles.someClass` but  through the  `autocompleteClasses` and `textFieldClasses`. They are not marked as `used` when applied this way.